### DISCUSSION TITLE
Incorporate user objects into the data stored in the repositories

### DISCUSSION THREAD

**Discussion by nicolaasburgers** [nicolaasburgers/definition-library-releases#25] [Ideas]
Raised in the meeting with Enzyme APD: add user objects (`.ghuser`) to the type of clusters that is able to be added to the library.  This could even be another plug-in, but if this is incorporated into this plug-in it would involve the plug-in inspecting the contents of the User Objects directory and making changes when the versions are updated, etc.

**Comment by jakubczajczynski** [comment]
Excellent idea. I saw people on the discourse forum quite frustrated about synchronizing User Objects between the team members. The Pancake plugin adds the ability to point to different folders for the User Objects, but this is without version control, etc.
